worlds/adventure/__init__.py

```diff
@@ -0,0 +1,355 @@
+import base64
+import copy
+import itertools
+import math
+import os
+from enum import IntFlag
+from typing import Any, ClassVar, Dict, List, Optional, Set, Tuple
+
+from BaseClasses import Entrance, Item, ItemClassification, MultiWorld, Region, Tutorial, \
+    LocationProgressType
+from Main import __version__
+from Options import AssembleOptions
+from worlds.AutoWorld import WebWorld, World
+from Fill import fill_restrictive
+from worlds.generic.Rules import add_rule, set_rule
+# from .Client import L2ACSNIClient  # noqa: F401
+from .Options import adventure_option_definitions, DragonRandoType, DifficultySwitchA, DifficultySwitchB
+from .Rom import get_base_rom_bytes, get_base_rom_path, AdventureDeltaPatch, apply_basepatch, \
+    AdventureAutoCollectLocation
+from .Items import item_table, ItemData, nothing_item_id, event_table, AdventureItem, standard_item_max
+from .Locations import location_table, base_location_id, LocationData, get_random_room_in_regions
+from .Offsets import static_item_data_location, items_ram_start, static_item_element_size, item_position_table, \
+    static_first_dragon_index, connector_port_offset, yorgle_speed_data_location, grundle_speed_data_location, \
+    rhindle_speed_data_location, item_ram_addresses, start_castle_values, start_castle_offset
+from .Regions import create_regions
+from .Rules import set_rules
+
+
+class AdventureWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up Adventure for MultiWorld.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["word_fcuk"]
```

Comment:
was the guide made by word_fcuk? They don't appear in the commit history

Reply:
No, I was bad and forgot to check what that parameter was for.